progress/react/icons/package.json

```diff
@@ -12,23 +12,32 @@
     },
     "prettier": "@pxblue/prettier-config",
     "peerDependencies": {
-        "@types/react": "^16.9.19",
-        "react": "^16.6.3"
+        "clsx": "^1.1.1",
+        "color": "^3.1.2",
+        "react": "^16.6.3",
+        "react-jss": "^10.4.0",
+        "react-dom": "^16.8.6"
```

Comment:
make clsx, color, and react-jss direct dependencies. We don't need to ensure a single instance of these like we do with material-ui or react.

Reply:
This makes sense, I've moved them from peerDependencies to dependencies, and deleted them from devDependencies.